docs/docs/modeling.md

```diff
@@ -118,4 +118,28 @@ classDiagram
 
 ## AssignedCategory -　
 
-## AssignedTask -
+## AssignedTask - 割当タスク
+
+### taskId
+
+- TaskId is the UUID
+
+### title
+
+- The owner can set a title as they prefer. This title can be modified at any time.
+
+Constraints:
+
+- It must be between 1 and 20 characters in length.
```

Comment:
I think you don't follow the notation rules in terms of the bullet point. Could you modify that?
https://scrapbox.io/tascurator/%F0%9F%8F%B7%EF%B8%8F_GitHub%E3%81%AE%E9%81%8B%E7%94%A8%E3%83%AB%E3%83%BC%E3%83%AB